app/utils/domain_resolver.py

```
@@ -97,50 +156,150 @@ def check_megakino_domain_validity(base_url: str, timeout: float | int = 15) ->
+                logger.debug("Megakino mirrors file returned HTML at {}", mirrors_url)
+                continue
+            domains = _parse_mirror_domains(resp.text or "")
+            if domains:
+                logger.info(
+                    "Megakino mirrors loaded from {} ({} entries)",
+                    mirrors_url,
+                    len(domains),
+                )
+                return domains
+            logger.debug("Megakino mirrors file empty at {}", mirrors_url)
+        except Exception as exc:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)